ArcadeLabsInc/arcade
# Update direct message screen

**Changes**:

- Use latest arclib
- Make sure direct messages is sorted by created_at
- Set hide nsec by default in login screen
